packages/calcite-components-react/src/auto-define.ts

```diff
@@ -0,0 +1,12 @@
+const isBrowser = (): boolean =>
```

Comment:
Suggestion (minor): The result of this can be computed earlier or cached. Multiple `autoDefine` calls run the same logic.

Reply:
Per our Teams conversation, we decided to leave this as is for now just to be very safe.

It is possible (but unlikely) that if we change isBrowser to a constant its could be cached on the server as `false` by an SSR framework trying to be too smart. If this were to happen and the constant didn't get reevaluated on the client, the components would never be defined.

This needs a bit more investigation.